### ISSUE TITLE
Create decorator to abstract route implementation

### ISSUE THREAD

**Issue by samihenrique** [vitta-health/attiv#43]

**Comment by taciomedeiros** [comment]
@samihenrique i like this one, can we describe better what is and create some tasklist for that? There is an inspiration in another framework that we could use?

**Comment by samihenrique** [comment]
> @samihenrique i like this one, can we describe better what is and create some tasklist for that? There is an inspiration in another framework that we could use?

Yes, I currently use this feature at https://github.com/samihenrique/express-js-kick-starter/tree/master/src/common/decorators

and it goes a long way, the idea is to create decorators to remove the amount of extra files we need to create to set things up in express, and bring it closer to what we already see in Spring and ASP.NET WebApi
